### PR TITLE
Monthly mood

### PR DESCRIPTION
**Pull request by Lmcourt**
# Change Log

- Closes #

## What I did:

- Update testing for monthly mood
- Update monthly mood to take the date from beginning of current month to end of current month

## Things to consider:

- 

## Test coverage

- [] 100% Model Testing
- [x] 100% Request Testing

If below 100%, what still needs testing:

-

### CODE REVIEW

**Review by TannerDale — APPROVED** [review]
LGTM

File: spec/factories/moods.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   factory :mood do
     association :user
 
-    mood { [1, 2, 3, 4, 5].sample }
+    mood { [0, 1, 2, 3, 4].sample }
```
Comment (WMudgeEllis) [inline spec/factories/moods.rb:5]: Front end will be sending back 1-5 as mood type (I think), can we change this to reflect that?

Front end chime in whenever you review this.

**Review by WMudgeEllis — COMMENTED** [review]
Looks good, but that factory may need to be changed back if front end is planning on sending back 1-5.

File: spec/factories/moods.rb (hunk 1 above)
Comment (TannerDale) [reply spec/factories/moods.rb:5]: Talking to Natalia she said 0-4 is fine

**Pull request merged**